tests/test_add.py

```diff
@@ -178,6 +178,14 @@ def test_otu_create_with_acronym_manual(self, precached_repo: Repo):
 
         assert otu.acronym == "FBNSV"
 
+    def test_otu_create_with_undelimited_segment_data_ok(self, precached_repo: Repo):
```

Comment:
Test the behaviour in `TestExtractSegmentNameFromRecord`. This test of `create_otu` is slow and only tests that OTU creation doesn't fail with undelimited names.